sigs.yaml

```diff
@@ -2474,3 +2428,51 @@ workinggroups:
     contact:
       slack: wg-k8s-infra
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-k8s-infra
+usergroups:
+  - name: Big Data
+    dir: ug-big-data
+    mission_statement: >
+      Serve as a community resource for advising big data and data science related software projects
+      on techniques and best practices for integrating with Kubernetes.
+      Represents the concerns of users from big data communities to Kubernetes for the purposes of
+      driving new features and other enhancements, based on big data use cases.
+    charter_link:
```

Comment:
@erikerlandson You can add the charter link here instead of specifying it in the custom content.

Reply:
ah, nice